TouchPortalAPI/client.py

```diff
@@ -122,7 +123,8 @@ def __init__(self, pluginId:str,
                  checkPluginId:bool = True,
                  updateStatesOnBroadcast:bool = True,
                  maxWorkers:int = None,
-                 executor:Executor = None):
+                 executor:Executor = None,
+                 callbackType:str = "Default"):
```

Comment:
Can we make this a bool True/False instead of a string lookup? Would be much faster to compare with in _emitEvent(). 

Could call it something like `useNamespaceCallbacks` or `useObjectsInCallbacks`? Or something better... :)

Reply:
Thanks. Fixed